website/config/routes.js

```diff
@@ -392,8 +392,16 @@ module.exports.routes = {
   'GET /handbook/digital-experience': '/handbook/marketing#digital-experience',
   'GET /handbook/digital-experience/article-formatting-guide': '/handbook/marketing/article-formatting-guide',
   'GET /handbook/digital-experience/commonly-used-terms': '/handbook/marketing/commonly-used-terms',
+  'GET /handbook/marketing/commonly-used-terms',: '/handbook/company/communications#commonly-used-terms',
   'GET /handbook/digital-experience/how-to-submit-and-publish-an-article': '/handbook/marketing/how-to-submit-and-publish-an-article',
   'GET /handbook/digital-experience/markdown-guide': '/handbook/marketing/markdown-guide',
+  'GET /handbook/marketing/markdown-guide': '/handbook/company/communications#writing-in-fleet-flavored-markdown', 
+  'GET /handbook/marketing/content-style-guide': '/handbook/company/communications#writing', 
+  'GET /handbook/marketing/editor-guide/': '/handbook/marketing/README#how-to-edit-markdown-pull-requests-for-the-docs', 
+  'GET /handbook/marketing/editor-guide#how-to-make-edits-with-github': '/handbook/company/communications#making-a-pull-request', 
+  'GET /handbook/marketing/editor-guide#how-to-edit-recently-merged-pull-requests-for-the-handbook': '/handbook/company/communications#editing-a-merged-pull-requests',
```

Comment:
putting fragments `#` on the left-hand side doesn't work.

Reply:
Eric or Rachael or I can show you how to run the website on localhost:2024 to test things